advanced/kernel-messaging/README.md

```diff
@@ -29,28 +29,36 @@ execution result.
 
 ## Initializing and managing a kernel session (`panel.ts`)
 
-Jupyterlab provides a class `ClientSession`
-([see the documentation](https://jupyterlab.github.io/jupyterlab/apputils/classes/clientsession.html))
+Jupyterlab provides a class `SessionContext`
+([see the documentation](https://jupyterlab.github.io/jupyterlab/apputils/classes/sessioncontext.html))
```

Comment:
This link doesn't work (yet). We should check whether it gets updated once 2.0 final is published.